### PR TITLE
Added more styling preferences, removed few issues with insets in xib

### PR DESCRIPTION
**Pull request by JakubJagoda**

### CODE REVIEW

File: VENTokenField/VENToken.m [hunk 1]
```diff
@@ -52,8 +51,23 @@ - (void)setTitleText:(NSString *)text
 {
     self.titleLabel.text = text;
     self.titleLabel.textColor = self.colorScheme;
+    [self updateTitleLabelFrame];
+}
+
+- (void)updateTitleLabelFrame {
+    //adjusting width also makes
```
Comment (JakubJagoda) [inline VENTokenField/VENToken.m]: Oh crap I left it in the middle of edition. I'll fix this now

File: VENTokenField/VENTokenField.m [hunk 2]
```diff
@@ -86,10 +85,12 @@ - (void)setUpInit
     self.horizontalInset = VENTokenFieldDefaultHorizontalInset;
     self.tokenPadding = VENTokenFieldDefaultTokenPadding;
     self.minInputWidth = VENTokenFieldDefaultMinInputWidth;
+    self.toLabelPadding = VENTokenFieldDefaultToLabelPadding;
     self.colorScheme = [UIColor blueColor];
     self.toLabelTextColor = [UIColor colorWithRed:112/255.0f green:124/255.0f blue:124/255.0f alpha:1.0f];
     self.inputTextFieldTextColor = [UIColor colorWithRed:38/255.0f green:39/255.0f blue:41/255.0f alpha:1.0f];
-    
+    self.font = [UIFont fontWithName:@"HelveticaNeue" size:15.5];
```
Comment (typeofweb) [inline VENTokenField/VENTokenField.m:92]: Why not `[UIFont systemFontOfSize:15.5f]`?

**Comment by grEvenX** [comment]
Good by me, but see @mmiszy 's comment about font...

File: VENTokenField/VENTokenField.m (hunk 2 above)
Comment (JakubJagoda) [inline VENTokenField/VENTokenField.m:92]: It's just a default value, consistent with the rest of the plugin. It'll be overriden by the element that will use this token field anyway ;)

Comment (typeofweb) [inline VENTokenField/VENTokenField.m:92]: Okay :)

**Comment by typeofweb** [comment]
We should probably update the demo at some point in the future :smile:

**Comment by JakubJagoda** [comment]
Hmm if I hadn't removed that line that made rounded corners, this demo wouldn't change anyhow :P

**Pull request merged**